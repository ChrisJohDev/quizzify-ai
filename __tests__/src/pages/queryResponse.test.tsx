import {render, screen }from '@testing-library/react';
import QueryResponse from '@/pages/queryResponse';
import '@testing-library/jest-dom'

describe('QueryResponse - snapshot test', () => {
  it('renders correctly', () => {
    const { container } = render(<QueryResponse />);
    expect(container).toMatchSnapshot();
  });
});