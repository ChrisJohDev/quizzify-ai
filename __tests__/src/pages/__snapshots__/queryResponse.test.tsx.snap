// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueryResponse - snapshot test renders correctly 1`] = `
<div>
  <h1>
    Query Response
  </h1>
</div>
`;
